hack/run-e2e-kind.sh

```diff
@@ -376,6 +376,50 @@ function setup-mcad-env {
 
 }
 
+function extend-resources {
+    # Patch nodes to provide GPUs resources without physical GPUs.
+    # This is intended to allow testing of GPU specific features such as histograms.
+
+    # Start communication with cluster
+    echo -n "Starting proxy "
+
+    kubectl proxy > /dev/null 2>&1 &
+    PROXY_PID=$!
+
+    echo "(pid=${PROXY_PID})..."
+
+    # Variables
+    RESOURCE_NAME="nvidia.com~1gpu"
+    RESOURCE_COUNT="8"
+
+    # Patch nodes
+    for NODE_NAME in $(kubectl get nodes --no-headers -o custom-columns=":metadata.name")
+    do
+        echo "- Patching node (add): ${NODE_NAME}"
+
+        curl --header "Content-Type: application/json-patch+json" \
+             --request PATCH \
+             --data '[{"op": "add", "path": "/status/capacity/'${RESOURCE_NAME}'", "value": "'${RESOURCE_COUNT}'"}]' \
+             http://localhost:8001/api/v1/nodes/${NODE_NAME}/status
+
+        echo
+    done
+
+    # Stop communication with cluster
+    echo "Killing proxy (pid=${PROXY_PID})..."
+    kill ${PROXY_PID}
+
+    # Run kuttl tests to confirm GPUs were added correctly
+    kuttl_test="${ROOT_DIR}/test/kuttl-test-extended-resources.yaml"
+    echo "kubectl kuttl test --config ${kuttl_test}"
+    kubectl kuttl test --config ${kuttl_test}
+    if [ $? -ne 0 ]
+    then
+      echo "kuttl e2e test '${kuttl_test}' failure, exiting."
+      exit 1
+    fi
+}
```

Comment:
Why not add this block to the array of kuttl tests at line 48 ?

Reply:
Because that function (line 48 and line 423) deletes the `quotasubtrees` at the end of test, of which I didn't create any. My tests were meant to be isolated from quota management, or MCAD for that matter. They make sure GPUs are added to the cluster and that is irrespective of whether quota is on or not, or if MCAD is ready or not. So that is why I believe it should be isolated.